README.md

```diff
@@ -26,12 +26,30 @@ $ sh build.sh
 
 ## Demo
 ### gauss_pdf_and_cdf.py
-Show and save {(gaussian, beta), (probability distribution function, cumulative distribution function)} figure.
 
-If you specify "gaussian" as the argument of the main function, show gaussian pdf and cdf,
+Show and save some distribution figures.
+
+The figures supported in this demo are shown at the follow table.
+|distribution type|function type|
+|:---|:---|
+|gaussian|PDF|
+|gaussian|CDF|
+|beta|PDF|
+|beta|CDF|
+
+PDF is probability distribution function and CDF is cumulative distribution function.
+
+- Run this demo. You can use command:
+```
+$ python src/gauss_pdf_and_cdf.py
+```
+
+If you specify "gaussian" as the argument of the main function, show gaussian PDF and CDF,
 ![gaussian_pdf](https://user-images.githubusercontent.com/49121951/116657667-c98e5100-a9c9-11eb-9651-891a6ae53ee5.png)
 ![gaussian_cdf](https://user-images.githubusercontent.com/49121951/116657716-d9a63080-a9c9-11eb-9671-2455ed4f6dd9.png)
 
-If you specify "beta" as the argument of the main function, show beta pdf and cdf,
+If you specify "beta" as the argument of the main function, show beta PDF and CDF,
 ![beta_pdf](https://user-images.githubusercontent.com/49121951/116657825-05291b00-a9ca-11eb-9172-18162e4ced07.png)
 ![beta_cdf](https://user-images.githubusercontent.com/49121951/116657853-14a86400-a9ca-11eb-86e1-d8f4077333a8.png)
+
+The figures are saved in figs/.
```

Comment:
Before creating PR, you should check yourself. In this case, you should check README's preview. What do you think about the following five points?
- English grammar is wrong.
- Definition of technical terms is written as plain texts without any emphasis. 
- Where figures are saved is written at the bottom.
- Figures are too big. You can show PDF and CDF in one line with two columns.
- Sub-sections aren't used. Make your documents easy for users to understand summary without reading sentences.